package.json

```diff
@@ -673,6 +673,17 @@
             "default": "true",
             "description": "%configuration.maven.dependency.enableConflictDiagnostics%",
             "scope": "window"
+          },
+          "maven.defaultOpenProjectMethod": {
+            "default": "Interactive",
```

Comment:
Do we have data for the favorite option? I prefer `Open` personally

Reply:
> Do we have data for the favorite option?

No we don't. We can add corresponding telemetry. Then we need to keep "interactive" as default for a while to collect data.

I'm 50:50, maybe prefer interactive a little bit. maybe my key point is, I don't think this notification is a frequent thing.

Reply:
Colleting data sounds good

Reply:
will sendInfo